types/minio/minio-tests.ts

```diff
@@ -106,8 +106,14 @@ minio.removeAllBucketNotification('testBucket');
 
 minio.listenBucketNotification('testBucket', 'pref_', '_suf', [ Minio.ObjectCreatedAll ]);
 
-minio.getBucketPolicy('testBucket', 'pref_', (error: Error|null, policy: Minio.PolicyValue) => { console.log(error, policy); });
-minio.getBucketPolicy('testBucket', '');
-
-minio.setBucketPolicy('testBucket', '', Minio.Policy.READWRITE, (error: Error|null) => { console.log(error); });
-minio.setBucketPolicy('testBucket', 'pref_', Minio.Policy.WRITEONLY);
+minio.getBucketPolicy('testBucket', (error: Error|null, policy: string) => { console.log(error, policy); });
+minio.getBucketPolicy('testBucket');
+
+const testPolicy = `{"Version":"2012-10-17","Statement":[{"Action":["s3:GetBucketLocation"],"Effect":"Allow",
```

Comment:
How can I get string like this? Is it result of JSON.stringify from newPostPolicy?

Reply:
Copied from minio-js [tests](https://github.com/minio/minio-js/blob/6c752fe370a49ac2efedc9b41e5e7e4591088772/src/test/functional/functional-tests.js#L631)